config/kccm/kustomization.yaml

```diff
@@ -14,6 +14,8 @@ patchesJson6902:
     name: kubevirt-cloud-controller-manager
 bases:
 - https://github.com/kubevirt/cloud-provider-kubevirt/config/isolated?ref=v0.3.2
+- ../rbac
+
```

Comment:
I think this is adding the rbac used to install capk to the template used to create a guest cluster with kccm